client/tests/integration/tx_history.rs

```diff
@@ -57,12 +57,12 @@ fn client_has_rejected_and_acepted_txs_should_return_tx_history() -> Result<()>
         .only_output();
     assert_eq!(transactions.len(), 50);
 
-    let mut prev_creation_time = 0;
+    let mut prev_create_time = core::time::Duration::from_millis(0);
     for tx in &transactions {
-        assert_eq!(&tx.payload().account_id, &account_id);
+        assert_eq!(tx.payload().authority(), &account_id);
         //check sorted
-        assert!(tx.payload().creation_time >= prev_creation_time);
-        prev_creation_time = tx.payload().creation_time;
+        assert!(tx.payload().create_time() >= prev_create_time);
```

Comment:
`creation_time` is more grammatically correct. But `create_time` is acceptable.

Reply:
silly change, I'll revert this

Reply:
we might consider `creation_timestamp` vs `creation_time`